### PR TITLE
add project representative field

### LINKED ISSUE #959: We need a separate project representative field
Currently the project owner name is used for the certificates but in QGIS I am the project owner but Paolo is the project rep. This leads to something bad: Paolo's signature with my name under it:

screen shot 2018-06-14 at 20 00 01

To fix please add a new field: Project representative and show it just above signature:

screen shot 2018-06-14 at 20 06 47

Then change the signature field name to 'Project representative signature' - and under it add the help text "This signature will be used on invoices and certificates"

### LINKED ISSUE #966: Chair name still Tim Sutton
While the signature is correctly taken from Paolo Cavallini the name on the certificates is still Tim Sutton

### PR DESCRIPTION
**Pull request by ann26**
fix #959 and fix #966 

This PR:
- [x] Rename field signature to project_representative_signature (your uploaded signature in this field won't be affected)
- [x] Add project_representative field
- [x] Change help text for project representative signature
- [x] Update project form
- [x] Change project owner name in the certificate to user project representative name
- [x] Update permissions for project representative

Form:
screenshot from 2018-07-02 09-42-31
peek 2018-07-02 09-45

Regenerate existing certificate will change the name of project representative:
peek 2018-07-02 09-47

### CODE REVIEW

File: django_project/base/models/project.py [hunk 1]
```diff
@@ -228,6 +229,16 @@ class Project(models.Model):
...
     )
 
+    project_representative = models.ForeignKey(
+        User,
+        related_name='project_representative',
+        help_text=_(
+            'Project representative. '
+            'This name will be used on invoices and certificates. '),
+        blank=True,
+        null=True # This is needed to populate existing database.
```
Comment (ismailsunni) [inline django_project/base/models/project.py]: __[Flake8]__

[E261] at least two spaces before inline comment

**Comment by timlinux** [comment]
LGTM - please merge when tests pass

File: django_project/certification/tests/views/test_certificate_views.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 import logging
 from mock import patch, MagicMock
 from django.core.urlresolvers import reverse
+from django.http import Http404
```
Comment (ismailsunni) [inline django_project/certification/tests/views/test_certificate_views.py]: __[Flake8]__

[F401] 'django.http.Http404' imported but unused

**Comment by timlinux** [comment]
@ann26 can you get this merged ASAP please?

File: django_project/certification/tests/views/test_certificate_views.py [hunk 3]
```diff
@@ -1,7 +1,13 @@
 # coding=utf-8
+from io import BytesIO
+from StringIO import StringIO
 import logging
 from mock import patch, MagicMock
+from PIL import Image
+from django.core.files.base import ContentFile
 from django.core.urlresolvers import reverse
+from django.core.files.images import ImageFile
```
Comment (ismailsunni) [inline django_project/certification/tests/views/test_certificate_views.py]: __[Flake8]__

[F401] 'django.core.files.images.ImageFile' imported but unused

File: django_project/certification/tests/views/test_certificate_views.py [hunk 4]
```diff
@@ -1,7 +1,13 @@
 # coding=utf-8
+from io import BytesIO
```
Comment (ismailsunni) [inline django_project/certification/tests/views/test_certificate_views.py]: __[Flake8]__

[F401] 'io.BytesIO' imported but unused

File: django_project/certification/tests/views/test_certificate_views.py [hunk 5]
```diff
@@ -1,7 +1,12 @@
 # coding=utf-8
+import tempfile
+from StringIO import StringIO
```
Comment (ismailsunni) [inline django_project/certification/tests/views/test_certificate_views.py]: __[Flake8]__

[F401] 'StringIO.StringIO' imported but unused

File: django_project/certification/tests/views/test_certificate_views.py [hunk 6]
```diff
@@ -1,7 +1,12 @@
...
 import logging
 from mock import patch, MagicMock
+from PIL import Image
+from django.core.files.base import ContentFile
```
Comment (ismailsunni) [inline django_project/certification/tests/views/test_certificate_views.py]: __[Flake8]__

[F401] 'django.core.files.base.ContentFile' imported but unused

**Comment by ann26** [comment]
hi @timlinux I merged this despite the codecov/patch hits 63.63% out of target 66.89% because I have tried several method to make it covers everything and it failed somehow. But the code should be fine.

**Pull request merged**